src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -7,9 +7,8 @@
 
     /* Prefix definitions */
     public static final Prefix PREFIX_NAME = new Prefix("n/");
-    public static final Prefix PREFIX_PHONE = new Prefix("p/");
-    public static final Prefix PREFIX_EMAIL = new Prefix("e/");
-    public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
+    public static final Prefix PREFIX_DESCRIPTION = new Prefix("d/");
+    public static final Prefix PREFIX_DEADLINE = new Prefix("d/");
```

Comment:
Would it hard the application distinguish the two fields since the use the same prefix? Suppose their order change, what would happen?